### PR TITLE
[SASS-2468] Fixed bugs

### PR DESCRIPTION
**Pull request by erasmos-hmrc**
# Checklist

Jira subtask: SASS-2468

## Developer: Before Asking Others To Review Your Code

Have you ...

- [x]  Had a QA verify that all acceptance criteria have been met, with comment(s) in the ticket? 
- [x]  Hosted a group review of your implementation?
- [x]  Rebased against `main`? 
- [x]  Ensured that your commit(s) contain the ticket key?
- [x]  Ensured that your branch is named after the ticket key?
- [x]  Successfully run the unit tests?
- [x]  Successfully run the integration tests?
- [x]  Successfully run the journey tests? If not, please add a comment explaining why it wasn't necessary.
- [x]  Ensured that the PR job has passed.

## Reviewer: Before Reviewing

Have you ...

- [x]  Pulled the branch down?
- [x]  Assigned yourself to this PR?
- [x]  Moved the Jira *subtask* to “in review”?
- [x]  Ensured that it is rebased against `main`?

## Developer: After Approval

Have you ...

- [x]  Rebased against `main`?
- [ ]  Moved the Jira *subtask* to “in pipeline”?

## Developer: After Merging

Have you ...

- [ ]  Deleted the branch?
- [ ]  Scheduled a demo? If not, please add a comment.

### CODE REVIEW

**Comment by salman-a-dev** [comment]
Requires a Rebase @erasmos-hmrc

File: app/controllers/pensions/unauthorisedPayments/UnauthorisedPaymentsCYAViewHelper.scala [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+package controllers.pensions.unauthorisedPayments
+
+import models.pension.charges.UnauthorisedPaymentsViewModel
+import play.api.i18n.Messages
+import play.api.mvc.Call
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import utils.ViewUtils
+
+object UnauthorisedPaymentsCYAViewHelper {
```
Comment (salman-a-dev) [inline app/controllers/pensions/unauthorisedPayments/UnauthorisedPaymentsCYAViewHelper.scala:26]: I would maybe have this in a helper directory 
From: [app/controllers/pensions/unauthorisedPayments/UnauthorisedPaymentsCYAViewHelper.scala]
To This: app/controllers/pensions/unauthorisedPayments/**helper**/UnauthorisedPaymentsCYAViewHelper.scala]

Comment (erasmos-hmrc) [reply app/controllers/pensions/unauthorisedPayments/UnauthorisedPaymentsCYAViewHelper.scala:26]: OK.

Comment (erasmos-hmrc) [reply app/controllers/pensions/unauthorisedPayments/UnauthorisedPaymentsCYAViewHelper.scala:26]: Though I think `helpers` would be more consistent.

Comment (erasmos-hmrc) [reply app/controllers/pensions/unauthorisedPayments/UnauthorisedPaymentsCYAViewHelper.scala:26]: I've changed my mind.

As discussed, let's keep it as is, and revisit when we add another similar helper.

**Review by salman-a-dev — APPROVED** [review]

**Pull request merged**